internal/v4/api_test.go

```diff
@@ -626,6 +609,19 @@ func (s *APISuite) TestMetaPerm(c *gc.C) {
 	c.Assert(e.Public, jc.IsFalse)
 	c.Assert(e.ACLs, jc.DeepEquals, mongodoc.ACL{})
 	c.Assert(e.ACLs.Read, gc.DeepEquals, []string{})
+
+	// Try setting all permissions in one request
```

Comment:
You can make a put with on a Read or a Write, right? Shouldn't you demonstrate that?